packages/app/src/templates/reducers/template.reducer.test.ts

```diff
@@ -38,3 +40,32 @@ describe("ReadAllIndexes data functionality", () => {
     })
   })
 })
+
+describe("Delete Template data functionality", () => {
+  const template: Template = {
+    id: "1",
+    text: "Test template",
+    lastUsedAt: "1574335694",
+  }
+
+  test("Event: DeleteTemplate update properly data field", () => {
```

Comment:
Just to be sure, it would be good to test what happens when in the store there are e.g. three templates with id 1,2,3 and you try to delete two of them, e.g. 1 and 3 (you could just extend this test).

Reply:
Test with more elements added in [ca0ccd5](https://github.com/mudita/mudita-center/pull/752/commits/ca0ccd5977773430798c01ef71f22f8086da8e6b)